### PR TITLE
Tutorial updates

### PR DESCRIPTION
**Pull request by mindyls**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#NNN

# Testing Instructions

# Summary

# Output

### CODE REVIEW

**Review by mindyls — COMMENTED** [review]
This completed successfully before changing the "base_test.sh" calls (so recenter_tc, data_fusion, template_basic_plugin should be good to go).

We may want to make a few more small changes before merging:
1. Only compare clean imagery in base_test.sh (run annotated/config-based, but do not compare outputs necessarily)
2. Update expected installation output

Package: full
Total run time: 2431 seconds
Number data types run: 38
Number data types failed: 0

File: docs/source/starter/installation.rst [hunk 1]
```diff
@@ -14,155 +14,435 @@
...
+
+    SUCCESS: 'rclone' appears to be installed successfully
+        /root/micromamba/envs/geoips/bin/rclone
+
+    Updated Git hooks.
+    Git LFS initialized.
+
+    **Cloning, updating, and uncompressing test_data_amsr2.git
+
+    **Cloning test_data_amsr2.git
```
[the remark this answers is not in the document]
Comment (jsolbrig) [reply docs/source/starter/installation.rst]: I reduced this to just showing the last few lines from `base_test.sh`.

File: tests/test_all.sh [hunk 2]
```diff
@@ -21,6 +21,16 @@
...
 
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh gitlfs
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh imagemagick
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh wget
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh git
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh python
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh rclone
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_clavrx
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_amsr2
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_abi_day
```
Comment (jsolbrig) [inline tests/test_all.sh:32]: Do we still install test_data_abi_day or is this superseded by test_data_noaa_aws?

**Review by jsolbrig — APPROVED** [review]

File: tests/test_all.sh (hunk 2 above)
Comment (mindyls) [reply tests/test_all.sh:32]: This has always put the abi data in test data noaa aws, that is where the test scripts point,  right?

**Pull request merged**